.github/workflows/build-and-test.yml

```diff
@@ -6,23 +6,10 @@ jobs:
   test:
     strategy:
       matrix:
-        stack: [heroku-20, heroku-22, heroku-24]
+        stack_version: [20, 22, 24]
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v3
-
-    - name: Run Buildpack Compile
-      run: |
-        ./bin/compile . . .
-      env:
-        STACK: ${{ matrix.stack }}
-
     - name: Check Installation
-      run: |
-        # Trick the profile script into using working dir as HOME
-        HOME="$(pwd)"
-        # Verify profile script puts Chrome & Chromedriver on the PATH
-        source .profile.d/chrome-for-testing.sh
-        chrome --version
```

Comment:
These `chrome --version` checks ensure that Chrome can start, but were being removed in this PR.

I've added them back in `bin/test.sh` along with some extra checks to ensure Chrome can boot. For test robustness I also switched to a multi-stage build for the development/CI `Dockerfile`, which builds the image against the Heroku build image, but then the app against the Heroku run image - to match what happens when people use this buildpack on the platform. See:
7b219e85e3ea4ce8460c0d49b65ca1f7b56aade3

Adding these tests back actually found that Chrome was still broken on Heroku-24, for example:

```
chrome: error while loading shared libraries: libasound.so.2: cannot open shared object file: No such file or directory
```

See:
https://github.com/heroku/heroku-buildpack-chrome-for-testing/actions/runs/9302921661/job/25604116446#step:3:576

I will add another commit now, that adds the missing package.

Reply:
I've fixed the missing package (and cleaned up the packages list to remove packages no longer required and/or that are already in the base image), in 58948696c2b3370b19d6c1bc101ab0bc0ebc907e.